base/access_provider/api.go

```diff
@@ -33,11 +36,22 @@ type AccessSyncResult struct {
 	Error *error2.ErrorResult
 }
 
+// AccessSyncConfig gives us information on how the CLI can sync access providers
+type AccessSyncConfig struct {
+	// SupportPartialSync if true, syncing only out of sync access providers is allowed
+	SupportPartialSync bool
+
+	// SupportDeleteOnNameSync if true, access providers can be deleted by name only
+	SupportDeleteOnNameSync bool
```

Comment:
This might not be obvious what this means to an external developer. Otoh if this has a default value hidden in the RBAC and ACL wrappers then this is ok